### PR TITLE
https://circleci.atlassian.net/browse/FLEETS-1198

### PR DESCRIPTION
**Pull request by sebastian-lerner**
https://circleci.atlassian.net/browse/FLEETS-1198

### CODE REVIEW

**Comment by sebastian-lerner** [comment]
update docs to account for new out-of-the-box support for non-POSIX shells

File: jekyll/_cci2/orb-author-faq.md [hunk 1]
```diff
@@ -92,17 +92,44 @@ CircleCI orbs package [CircleCI reusable config]({{site.baseurl}}/reusing-config
...
+
+```yaml
+steps:
+ - run:
+    name: Check Node shell
+    shell: node
+    command: console.log("node")
+ ```
+ 
+  **Node**
```
Comment (pete-woods) [inline jekyll/_cci2/orb-author-faq.md:115]: You've got node twice

Comment (sebastian-lerner) [reply jekyll/_cci2/orb-author-faq.md:115]: ugh fixing one sec

File: jekyll/_cci2/orb-author-faq.md [hunk 2]
```diff
@@ -92,17 +92,44 @@ CircleCI orbs package [CircleCI reusable config]({{site.baseurl}}/reusing-config
...
+     shell: node
+     command: console.log("node")
+```
+
+  **Python**
 
 ```yaml
 steps:
   - run:
-    shell: /usr/bin/python3
```
Comment (pete-woods) [inline jekyll/_cci2/orb-author-faq.md:102]: 👍

**Review by devinmarieb — APPROVED** [review]

**Pull request merged**